brjs-sdk/sdk/libs/javascript/br-presenter/src/br/presenter/control/selectionfield/JQueryAutoCompleteControl.js

```diff
@@ -163,4 +161,12 @@ JQueryAutoCompleteControl.prototype.setOptions = function(mOptions) {
 	}
 };
 
+/**
+ * Destroy created listeners and jQuery autocomplete plugin
+ */
+JQueryAutoCompleteControl.prototype.destroy = function() {
+	this.m_oPresentationNode.value.removeListener(this._valueChangedListener);
```

Comment:
Yes, Today we found out that not all Nodes that use jQueryAutoCompleteControl had triggered `onViewReady` and then it will break the execution because the `autocomplete` plugin will throw error as no autocomplete was set.